vmm/modules/vtd/vtd.c

```diff
@@ -147,18 +152,23 @@ static void vtd_calculate_trans_cap(uint8_t *max_leaf, uint8_t *tm, uint8_t *sno
 		"VT-d: 3/4-level page-table is NOT supported!\n");
 }
 
-static void vtd_send_global_cmd(vtd_engine_t *engine, uint32_t cmd)
+static void vtd_send_global_cmd(vtd_engine_t *engine, VTD_GCMD_BIT_OFF off, boolean_t set)
```

Comment:
rename:
VTD_GCMD_BIT_OFF -> VTD_GCMD_BIT. OFF is confusing. In spec, the name is "Bits". So just use BIT here.
off -> bit.
boolean_t set -> uint32_t value. by using "value", you will put 0/1 there, which is more readable than True/False